src/plugins/lua/README.md

```diff
@@ -5,21 +5,82 @@
 - infos/needs =
 - infos/placements =
 - infos/status = maintained unittest global nodoc
-- infos/description = magic things require magic plugins
+- infos/description = proxy that calls other plugins (scripts) written in lua
 
 ## Introduction ##
 
-The plugin uses Lua to do magic things.
+The plugin uses Lua to do magic things. It basically allows to call plugins written in Lua.
+
+What a Lua script can do is not really limited by design, so any kind of plugin may be
+implemented. The lua plugin is especially useful to write filter and logging scripts.
 
 ## Usage ##
 
-    kdb mount file.ini /lua ini lua script=/path/to/lua/lua_filter.lua
+The lua plugin accepts only the **script** configuration parameter holding the path to a Lua
```

Comment:
Can you mention here that tree.libelektra.org/src/plugins/lua/lua/ contains examples?